lte/gateway/c/core/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -610,3 +617,45 @@ static void mme_app_exit(void) {
   OAI_FPRINTF_INFO("TASK_MME_APP terminated\n");
   pthread_exit(NULL);
 }
+
+static void start_state_hashtable_timer(void) {
+  state_hashtable_timer_id = start_timer(
+      &mme_app_task_zmq_ctx, 1000 * state_hashtable_timer_sec,
+      TIMER_REPEAT_FOREVER, handle_state_hashtable_timer, NULL);
+}
+
+static int handle_state_hashtable_timer(zloop_t* loop, int id, void* arg) {
```

Comment:
same comment as in other handlers, we are just interested in the number of elements stored in the hash table.